lib-release.js

```diff
@@ -326,7 +326,9 @@ const confirmedRelease = (type, version) => {
           Step 2 complete...
         `));
     }).catch((reason) => {
-      if (!~reason.stdout.indexOf('working tree clean')) {
```

Comment:
@LucaMele 
@mjenny 
Seems that `stdout` isn't equally populated across platforms.
So I check for both `message` and `stdout`.

Still needs windows test.